ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-v1.2__49.yaml

```diff
@@ -276,6 +276,8 @@ tests:
         name: knative-eventing-kafka-broker-dispatcher
       - env: KNATIVE_EVENTING_KAFKA_BROKER_KAFKA_CONTROLLER
         name: knative-eventing-kafka-broker-kafka-controller
+      - env: KNATIVE_EVENTING_KAFKA_BROKER_POST_INSTALL
+        name: knative-eventing-kafka-broker-post-install
```

Comment:
There is one more missing in 4.10 that I added in https://github.com/openshift/release/pull/28016